### ISSUE TITLE
2019-10 RC3: MPIRUN Printing of PRT and DBG

### ISSUE THREAD

**Issue by OPMUSER** [OPM/opm-simulators#2147]
Running 2019-10-RC3 under Ubuntu 18.04 LTS using:

`
mpirun -np 2  flow --parameter-file=MOD03-HRZ01-OPM1910-R01.param
`
 
I get additional *.PRT and *.DBG files namely:

```
MOD03-HRZ01-OPM1910-R01.PRT
MOD03-HRZ01-OPM1910-R01.DBG

MOD03-HRZ01-OPM1910-R01.1.DBG
MOD03-HRZ01-OPM1910-R01.1.PRT
```
This job stops due to numerical errors, with the first set of files being complete but the second set being incomplete. Is it it possible to cleanup after this occurs? As if some one is running with 64 nodes there will be 64 *.DBG and 64 *.PRT files created.

**Comment by blattms** [comment]
This is kind of a known issue. These files are created to capture logging fallout in parallel runs. That happens if logging is not done only on the master processor and is there to help debugging in this case.
For a normally ending simulation these files should be empty and cleaned up at the end of the run. If they have text in them it will appended to the expected files (without the .[1-N].).

In your case these files should actually have been cleaned up. 

But if you manually kill a run (or an uncaught exception is thrown) then these files will not be removed and you will see them after the run. If you run flow again for the deck and the simulation succeeds then they will be gone again. (Just tested this for the release and SPE1.)

I would like to postpone this to the next release.

**Comment by OPMUSER** [comment]
> I would like to postpone this to the next release.

That's okay by me, it's just an annoyance rather than a bug.

**Comment by OPMUSER** [comment]
Close.
